src/components/tour/tour_step_indicator.tsx

```diff
@@ -93,7 +85,7 @@ export const EuiTourStepIndicator: FunctionComponent<EuiTourStepIndicatorProps>
       values={{ status, number }}
     >
       {(ariaLabel: string) => (
-        <li className={classes} aria-label={ariaLabel} {...rest}>
+        <li css={[styles]} className={classes} aria-label={ariaLabel} {...rest}>
```

Comment:
for my own edification: is the array syntax necessary? IIRC ```css={css`display: inline-block`}``` is valid, so couldn't you just do
```suggestion
        <li css={styles} className={classes} aria-label={ariaLabel} {...rest}>
```